seattle_example/extra_data.py

```diff
@@ -25,7 +25,8 @@
         Column('actual_upload', Float),
         Column('min_rtt', Integer),
         Column('location_type', String),
+	Column('isp_user',String),
         Column('client_ip', BigInteger),
-        Column('cost_of_service', Integer))
+        Column('cost_of_service', String))
```

Comment:
I'll correct that indentation issue now.

For the other items, I think this is a matter of me not outlining specifically which changes are being made where. Those two should have been separate bullets. Below is a more accurate summary:

- Correct data type for cost of service: [seattle_example/extra_data.py L30](https://github.com/opentechinstitute/piecewise/blob/seattle-inplace-patches/seattle_example/extra_data.py#L30) - since the cost is being stored as text, not as and integer.
- Convert fields that are decimal to float: [piecewise/piecewise/wsgi.py L73-75](https://github.com/opentechinstitute/piecewise/blob/seattle-inplace-patches/piecewise/piecewise/wsgi.py#L73)
- Add ISP field (detected) to complement isp_user which is entered in the extra_data form: [seattle_example/extra_data.py +L28](https://github.com/opentechinstitute/piecewise/blob/seattle-inplace-patches/seattle_example/extra_data.py#L28) and [collector/collector/wsgi.py +L55](https://github.com/opentechinstitute/piecewise/blob/seattle-inplace-patches/collector/collector/wsgi.py#L55)
- Also corrects the label for the MaxUpload stat aggregation: [piecewise/piecewise/aggregate.py L705](https://github.com/opentechinstitute/piecewise/blob/seattle-inplace-patches/piecewise/piecewise/aggregate.py#L705)